doc/sphinx/cp/polycrystal.rst

```diff
@@ -0,0 +1,10 @@
+Polycrystal models
+==================
+
+This python file provides polycrystal homogenization models.
+Here the goal is to homogenize the response of a large collection of single
```

Comment:
Is there a necessity to check for compatibility among grains when assuming Taylor homogenization ?

Reply:
The Taylor model doesn't preserve compatibility.  We could add more sophisticated options in the future if we need them.